ssl/ssl_conf.c

```diff
@@ -471,27 +472,39 @@ static int do_store(SSL_CONF_CTX *cctx,
         if (*st == NULL)
             return 0;
     }
-    return X509_STORE_load_locations(*st, CAfile, CApath) > 0;
+    return X509_STORE_load_file(*st, CAfile)
+        || X509_STORE_load_path(*st, CApath)
+        || X509_STORE_load_store(*st, CAstore);
```

Comment:
This needs changing as per the other similar changes

Reply:
Ah, yeah, thanks

Reply:
Done